include/swift/AST/Identifier.h

```diff
@@ -395,6 +445,16 @@ class DeclName {
   StringRef getString(llvm::SmallVectorImpl<char> &scratch,
                       bool skipEmptyArgumentNames = false) const;
 
+  /// Return the string representation of the name
+  StringRef str() const {
+    switch (getKind()) {
+    case DeclNameKind::Normal:
+      return getBaseName().str();
```

Comment:
Renaming sounds like the way to go since IIRC this behaviour is needed in quite a few places.